Apigee/ManagementAPI/Developer.php

```diff
@@ -408,18 +451,21 @@ protected static function loadFromResponse(Developer &$developer, array $respons
         $developer->firstName = $response['firstName'];
         $developer->lastName = $response['lastName'];
         $developer->userName = $response['userName'];
-        $developer->organizationName = $response['organizationName'];
         $developer->status = $response['status'];
         $developer->attributes = array();
         if (array_key_exists('attributes', $response) && is_array($response['attributes'])) {
-            foreach ($response['attributes'] as $attribute) {
-                $developer->attributes[$attribute['name']] = @$attribute['value'];
+            foreach ($response['attributes'] as $key => $attribute) {
+                if (is_array($attribute)) {
+                    $developer->attributes[$attribute['name']] = @$attribute['value'];
```

Comment:
What kind of errors needs to be suppressed here?  The only that I can imagine is the 'value' index does not exist which should not happen.